data/file.go

```diff
@@ -56,8 +56,10 @@ func (f *File) IsPicture() bool {
 		if strings.HasSuffix(strings.ToLower(f.Name()), ending) {
 			// Then check MIME-type
 			if fileType, err := f.getFileContentType(); err == nil {
-				f.Close()
-				return strings.HasPrefix(fileType, "image/")
+				defer f.Close()
```

Comment:
Calling `Close()` with `defer`. If `Close()` should fail and return an error (which is ignored here), the program's behaviour would be unclear. This way, the result will be returned reliably even if closing the file fails.

Note that this should not be done with writing access.